libcxx/test/std/atomics/atomics.ref/test_helper.h

```diff
@@ -45,12 +45,12 @@ template <class T, class StoreOp, class LoadOp>
 void test_seq_cst(StoreOp store_op, LoadOp load_op) {
```

Comment:
Should we introduce `using Unqualified = std::remove_cv_t<T>` here like we do in other places?